app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    com_spl = command.split()
+    if len(com_spl) != 3:
+        return
+    cmd, file_old, path_file = com_spl
+    if path_file.find("/") > -1 and os.path.exists(path_file) is False:
```

Comment:
use os.path.split
instead of checking find("/")
For now, your code is not cross-platform
Did you read the checklist???
https://github.com/vladimir-byshevets/py-move-file/blob/develop/checklist.md

Reply:
os.path.split() Метод в Python используется для разделения имени пути на пару голов и хвостов, find - используется для поиска символа, объясните в чём ошибка?